app/api/connectors/httpparsers/StandardIfsHttpParser.scala

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Is this a standard file copied from another API? If so, should there be a spec file to go along with it?